### DISCUSSION TITLE
clarification on "The API is returning gridpoints that are off by 1,1."

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#592] [Q&A] [answered]
I know I'm overcomplicating this, but I just want to make sure I understand the issue correctly for offices which are impacted...

Does the issue with "The API is returning gridpoints that are off by 1,1" mean that for a given set of GPS points using the `https://api.weather.gov/points/{points}` endpoint, that the **gridX**, **gridY** values returned are off by 1,1?

Or does it mean that gridpoints used in `https://api.weather.gov/gridpoints/{wfo}/{x},{y}` return a **bounding box\polygon** that is a 1,1 off?

Another way to ask the same question... which of these endpoints is giving the incorrect data?
https://api.weather.gov/points/{points}
or 
https://api.weather.gov/gridpoints/{wfo}/{x},{y}

I'm assuming it's the points/{endpoints} but I figured best to ask. 

And I guess another question, is there a way to return all the bounding boxes and the corresponding (correct) polygon?

**Comment by sullynole** [comment, marked as answer]
The gridX and gridY values returned from the /points/{lat,lon} endpoint are off by +1,+1 for the affected WFOs, thus the /gridpoints url returned from the /points endpoint has to be adjusted to get the exact grid x,y at that specific lat,lon value.
